coordinator/src/services/translation/cancellation/cancellation-practitioner-role.ts

```diff
@@ -16,7 +16,7 @@ export function createPractitionerRole(
     practitioner: createReference(practitionerReference),
     organization: createReference(organizationReference),
     code: createJobRoleNameCode(hl7AgentPerson.code._attributes.code),
-    telecom: toArray(hl7AgentPerson.telecom)[0]._attributes ? convertTelecom(hl7AgentPerson.telecom) : undefined
+    telecom: toArray(hl7AgentPerson.telecom)[0]?._attributes ? convertTelecom(hl7AgentPerson.telecom) : undefined
```

Comment:
I don't think that question mark is going to do anything. You'll still get an index out of bounds error if the array is empty

Reply:
Well looks like I'm wrong

Reply:
JS is weird yo